ci/jenkins/templates/conformance-template.yaml

```diff
@@ -25,6 +25,9 @@
               name: SONOBUOY_VERSION 
               default: '{sonobuoy_version}'
               description: Version of sonobouy image to use
+        - string:
```

Comment:
I see now. :thinking: what do you plan on filtering here?

By that I mean how do you plan on using this in CI? I feel like this is more of a developers/debugging tool if you know you have a failure.